src/cli.ts

```diff
@@ -50,10 +51,21 @@ const logToolbeltVersion = () => {
   log.debug(`Toolbelt version: ${pkg.version}`)
 }
 
+const validToken = (): boolean => {
+  const token = getToken()
+  if (!token) { return false }
+
+  const decoded = decode(token)
+  if (decoded === null || typeof decoded === 'string' || decoded.exp === undefined) { return false }
```

Comment:
no need to check for `decoded.exp`, `undefined < somethingDefined` is always `false`.
`!decoded` is somewhat more accurate than `decoded === null`, but :thisisfine:

Reply:
I think the pattern in JS is to use `!value` or `value` to check if `=== null` or `!=== null`.

So **nit**: `if (!decoded || !decoded.exp) return false;` (if the object has an `exp` property, it's not a `string` either)

Reply:
I think there is no need for manual checking at all. The route `https://vtexid.vtex.com.br/api/vtexid/pub/authenticated/user?authToken={{authToken}}` does all the validation we need, returning `null` on invalid tokens. This is also good because we rely on VTEX ID instead of our own design.

Reply:
Downside of that is doing an asynchronous request before every toolbelt command, effectively reducing the responsiveness of the CLI. Or is this function called only for `vtex local token` command?

Reply:
This is called before pretty much every CLI command

Reply:
I think it would be best not to do the remote call, the only validation I care about here is if I can read the expiration date

Reply:
Good point. If this is meant to be executed before every toolbelt command, there is no point in checking with VTEX ID, all APIs called will do that anyway.